lib/src/child-writer/index.js

```diff
@@ -56,20 +58,46 @@ const EXIT_CODES = require('../exit-codes');
 exports.write = (image, drive, options) => {
   const emitter = new EventEmitter();
 
-  utils.getTemporaryLogFilePath().then((logFile) => {
-    const argv = utils.getCLIWriterArguments({
-      entryPoint: rendererUtils.getApplicationEntryPoint(),
-      logFile: logFile,
-      image: image,
-      device: drive.device,
-      validateWriteOnSuccess: options.validateWriteOnSuccess,
-      unmountOnSuccess: options.unmountOnSuccess
-    });
+  const argv = utils.getCLIWriterArguments({
+    entryPoint: rendererUtils.getApplicationEntryPoint(),
+    image: image,
+    device: drive.device,
+    validateWriteOnSuccess: options.validateWriteOnSuccess,
+    unmountOnSuccess: options.unmountOnSuccess
+  });
+
+  // There might be multiple Etcher instances running at
+  // the same time, therefore we must ensure each IPC
+  // server/client has a randomly generated name.
```

Comment:
20-random-bytes is 160-bit 'security' ;-)
Perhaps using a PID or time value would be too easy for other (malicious) processes to guess? \*shrug\*